packages/fault-detector/src/helpers.ts

```diff
@@ -117,7 +121,12 @@ export const updateOracleCache = async <TSubmissionEventArgs>(
 
       // We've failed 3 times in a row, we're probably stuck.
       if (failures >= 3) {
```

Comment:
Likely never becomes relavent but likely good to at least anchor at the alchemy limit
`
```suggestion
      if (failures >= 3 && step < 2000) {
```